services/gateway/util/src/main/java/org/eclipse/ditto/services/gateway/util/config/security/DefaultOAuthConfig.java

```diff
@@ -123,32 +125,33 @@ public String toString() {
     }
 
     private static class SubjectIssuerCollector
-            implements Collector<Map.Entry<String, ConfigValue>, Map<SubjectIssuer, String>, Map<SubjectIssuer,
-            String>> {
+            implements Collector<Map.Entry<String, ConfigValue>, Map<SubjectIssuer, SubjectIssuerConfig>, Map<SubjectIssuer,
+                    SubjectIssuerConfig>> {
 
         private static SubjectIssuerCollector toSubjectIssuerMap() {
             return new SubjectIssuerCollector();
         }
 
         @Override
-        public Supplier<Map<SubjectIssuer, String>> supplier() {
+        public Supplier<Map<SubjectIssuer, SubjectIssuerConfig>> supplier() {
             return HashMap::new;
         }
 
         @Override
-        public BiConsumer<Map<SubjectIssuer, String>, Map.Entry<String, ConfigValue>> accumulator() {
+        public BiConsumer<Map<SubjectIssuer, SubjectIssuerConfig>, Map.Entry<String, ConfigValue>> accumulator() {
             return (map, entry) -> map.put(SubjectIssuer.newInstance(entry.getKey()),
-                    entry.getValue().unwrapped().toString());
+                    // TODO: I don't know how to go back from a ConfigValue to a Config
+                    DefaultSubjectIssuerConfig.of(entry.getValue().atPath(entry.getKey()), entry.getKey()));
```

Comment:
```suggestion
                    DefaultSubjectIssuerConfig.of(ConfigFactory.empty().withFallback(entry.getValue())));
```
This could/should work :)